src/vs/workbench/contrib/files/browser/explorerService.ts

```diff
@@ -33,6 +33,7 @@ export class ExplorerService implements IExplorerService {
 	private readonly disposables = new DisposableStore();
 	private editable: { stat: ExplorerItem, data: IEditableData } | undefined;
 	private _sortOrder: SortOrder;
+	private _lexicographicOptions: LexicographicOptions;
```

Comment:
Does it make sense to instead have one attribute that merges `_sortOrder` and `_lexicographicOptions` into one

Reply:
I'm having trouble seeing if the suggested change would make the code more readable or more maintainable. If I were to replace line 35 and 36 with  `private _sortOrderConfiguration: ISortOrderConfiguration;` (which is what I thin you're suggesting?) that would mean that a bunch of other places in the file I'd need to refer to `this._sortOrderConfiguration.sortOrder` and `this._sortOrderConfiguration.lexicographicOptions`... which makes the code longer and less readable I think. 

I guess you could argue that this change might encourage maintainers pay attention to both configuration settings when making changes related to file sort order - but also these are private attributes so that would only apply within this file. 

Would you be okay to leave this code as is? 

If not, or if you are suggesting something different, let me know and I can go ahead and make the change.

Reply:
I am ok with leaving the code as it is, thanks for considering my suggestion.